src/W/Popover.elm

```diff
@@ -145,6 +153,12 @@ over =
     Attribute <| \attrs -> { attrs | over = True }
 
 
+{-| -}
+dismissOnClick : Bool -> Attribute msg
+dismissOnClick v =
```

Comment:
I think we can make it so it doesn't receive a value at all! Just `dismissOnClick`.

If people want to make it conditional they can use `noAttr` as an alternative :)